### ISSUE TITLE
 Stellar overlay: Mandatory Flow control in bytes

### ISSUE THREAD

**Issue by pieterjan84** [pendulum-chain/spacewalk#549]
FYI: https://github.com/stellar/stellar-core/pull/4353

Flow control in bytes is now required. Also listed as breaking change in latest pre-release of stellar-core. 

Implementation made for Stellarbeat: https://github.com/stellarbeat/js-stellar-node-connector/blob/master/src/connection/flow-controller.ts

If I'm mistaking that this is an issue for your project, please ignore :-)

**Comment by prayagd** [comment]
Thank you @pieterjan84 for raising the issue.

cc @pendulum-chain/devs is this relevant for us?

**Comment by b-yap** [comment]
Yes it is an issue:
```
Error occurred during processing xdr message: ConfigError("Error{ code:ErrConf message:flow control bytes disabled }")
```

Thank you once again @pieterjan84 !

**Comment by ebma** [comment]
This was fixed in #550 but we forgot to close it.
